community/index/src/test/java/org/neo4j/index/internal/gbptree/SimpleLongLayout.java

```diff
@@ -29,21 +29,75 @@
 {
     private final int keyPadding;
     private String customNameAsMetaData;
+    private final boolean fixedSize;
+    private final int identifier;
+    private final int majorVersion;
+    private final int minorVersion;
 
-    SimpleLongLayout( int keyPadding, String customNameAsMetaData )
+    static class Builder
```

Comment:
Yes! This makes it so much easier to use. Just what a test utility class needs :D